### ISSUE TITLE
Windows Bug - Failing on Yext Pages Generate

### ISSUE THREAD

**Issue by lambdaFun94** [yext/pages#176]
### Describe the bug

I have tried by updating the package "@yext/pages" with latest version "1.0.0-beta.2" updated 16 hours ago in package,
while running command "yext pages generate" we are getting following issue.

### Reproduction

![image](https://user-images.githubusercontent.com/61638671/186471491-7c4f925a-dc0a-428a-9f6f-d24dad5056f4.png)

### System Info

```shell
System:
    OS: Windows 10 10.0.19044
    CPU: (4) x64 Intel(R) Core(TM) i5-4460  CPU @ 3.20GHz
    Memory: 4.83 GB / 15.87 GB
  Binaries:
    Node: 17.9.1 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.19 - C:\Program Files (x86)\Yarn\bin\yarn.CMD
    npm: 8.11.0 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (104.0.1293.47)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    @yext/pages: 1.0.0-beta.2 => 1.0.0-beta.2
```

### Used Package Manager

npm

### Used Browser

Edge

### Validations

- [X] Check that there isn't [already an issue](https://github.com/yext/pages/issues) that reports the same bug to avoid creating a duplicate.

**Comment by mancusi** [comment]
If possible, can you share the `package-lock.json` for this repository as well? This will help me determine which exact version is installed.

This exact issue should have been resolved by #173 so I want to confirm that the correct version is being used before investigating further.

**Comment by mancusi** [comment]
[This](https://github.com/yext/pages/blob/20a94330767c19705304bf65699e1ff10c0fcf1c/packages/pages/src/vite-plugin/build/closeBundle/moduleLoader.ts#L22) line in particular should've resolved this issue
